pkg/sql/alter_database.go

```diff
@@ -505,15 +514,6 @@ func (n *alterDatabasePrimaryRegionNode) switchPrimaryRegion(params runParams) e
 		return err
 	}
 
-	// Get the type descriptor for the multi-region enum.
-	typeDesc, err := params.p.Descriptors().GetMutableTypeVersionByID(
-		params.ctx,
-		params.p.txn,
-		n.desc.RegionConfig.RegionEnumID)
-	if err != nil {
-		return err
-	}
-
```

Comment:
This needed to be moved a tiny bit further up or else there would be cross-reference validation failures!